android/src/main/java/com/swmansion/rnscreens/SearchViewFormatter.kt

```diff
@@ -37,4 +45,27 @@ class SearchViewFormatter(var searchView: SearchView) {
             searchTextPlate.background = currentDefaultTintColor
         }
     }
+
+    fun setHeaderIconColor(headerIconColor: Int?) {
+        headerIconColor?.let {
+            searchIcon.setColorFilter(headerIconColor)
+            searchCloseIcon.setColorFilter(headerIconColor)
+        }
+    }
+
+    fun setTextHintColor(textHintColor: Int?) {
+        textHintColor?.let {
+            searchEditText?.setHintTextColor(textHintColor)
+        }
+    }
+
+    fun setHintSearchIcon(hintSearchIcon: Boolean?, placeholder: String?) {
```

Comment:
What do you think about changing this to `setPlaceholder(placeholder: String?, shouldShowHintSearchIcon: Boolean)`?

Reply:
I changed it !